org.virtualxt.VirtualXT.yml

```diff
@@ -0,0 +1,88 @@
+command: virtualxt.freedos
+finish-args:
+  #- --share=network
+  - --share=ipc
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --device=all
+build-options:
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: SDL2
+    buildsystem: autotools
+    config-opts:
+      - --disable-video-wayland
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: archive
+        url: https://www.libsdl.org/release/SDL2-2.0.20.tar.gz
+        sha256: c56aba1d7b5b0e7e999e4a7698c70b63a3394ff9704b5f6e1c57e0c16f04dd06
+
+    #modules:
+    #  - name: libdecor
+    #    config-opts:
+    #      - -Ddemo=false
+    #    buildsystem: meson
+    #    cleanup:
+    #      - /include
+    #      - /lib/pkgconfig
+    #    sources:
+    #      - type: archive
+    #        url: https://gitlab.gnome.org/jadahl/libdecor/uploads/81adf91d27620e20bcc5f6b9b312d768/libdecor-0.1.0.tar.xz
+    #        sha256: fdefa11de4bd51cb14223a97e41fdd848f01f5c5ddca9b036a0c4e3e74d9f486
+
+  #- name: pcap
+  #  buildsystem: autotools
+  #  sources:
+  #    - type: archive
+  #      url: https://github.com/the-tcpdump-group/libpcap/archive/refs/tags/libpcap-1.10.1.tar.gz
+  #      sha256: fdefa11de4bd51cb14223a97e41fdd848f01f5c5ddca9b036a0c4e3e74d9f486
+
+  - name: virtualxt
+    buildsystem: simple
+    build-commands:
+    
+      # Fix for 0.6.1. Remove later.
+      - cp _go.mod go.mod
+      - cp _go.sum go.sum
+      - cp _modules.txt vendor/modules.txt
+
+      - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/lib
+      - export C_INCLUDE_PATH=$C_INCLUDE_PATH:/include
+      #- $GOROOT/bin/go build -tags sdl,pcap
+      - $GOROOT/bin/go build -tags sdl
+
+      - install -Dm00755 virtualxt $FLATPAK_DEST/bin/virtualxt
+      - install -Dm00644 bios/pcxtbios.bin $FLATPAK_DEST/share/virtualxt/pcxtbios.bin
+      - install -Dm00644 bios/vxtx.bin $FLATPAK_DEST/share/virtualxt/vxtx.bin
+      - install -Dm00644 boot/freedos.img $FLATPAK_DEST/share/virtualxt/freedos.img
+      - install -Dm00644 boot/freedos_hd.img $FLATPAK_DEST/share/virtualxt/freedos_hd.img
+      - install -Dm00644 doc/manual/index.md.html doc/manual/markdeep.min.js doc/manual/style.css -t $FLATPAK_DEST/share/virtualxt/manual
+
+      - install -Dm00644 doc/icon/icon.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      
+      - install -Dm00755 virtualxt.freedos $FLATPAK_DEST/bin/virtualxt.freedos
+      - install -Dm00644 $FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm00644 $FLATPAK_ID.appdata.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+    sources:
+      - type: archive
+        url: https://github.com/andreas-jonsson/virtualxt/archive/refs/tags/v0.6.1.tar.gz
+        sha256: 6619e16925ab488bc7ee09852d05c98f758050aef9c115eec70a283c9bdc04dc
+
+      - type: dir
+        path: .
```

Comment:
you should not have a `dir` source. If it is for the various metadata file add them individually. Also they should be submitted upstream so that in the future they don't need to be here.

Reply:
They are now individual files.
Fixes are pushed upstream.